docs/introduction/understanding-docker.md

```diff
@@ -119,7 +119,7 @@ Docker images are the **build** component of Docker.
 Docker registries hold images. These are public or private stores from which you
 upload or download images. The public Docker registry is provided with the
 [Docker Hub](http://hub.docker.com). It serves a huge collection of existing
-images for your use. These can be images you create yourself or you can use
+images for your use. These can be images you created for yourself or 
```

Comment:
Agree. This does not need to change. 

Aside: I would rewrite that sentence as "These can be images that you create or images that others have created." Although that reads as tautological. Either you made the images or someone else did. What's trying to be said is that a user can upload their own images for reuse, as well as reusing images provided by others.